robolectric-shadows/shadows-core/src/main/java/org/robolectric/fakes/RoboMenu.java

```diff
@@ -21,7 +23,7 @@
   private Context context;
 
   public RoboMenu() {
-    this(null);
+    this(RuntimeEnvironment.application);
```

Comment:
Kill this constructor, its clear that a Context is needed, its better to make that obvious in the API rather than defaulting it here. As I said earlier, with different contexts you can have different resources.